dasharo/README.md

```diff
@@ -65,3 +65,140 @@ pub   rsa4096 2021-11-10 [SC] [expires: 2022-11-10]
 uid                      <Platform Name> <Platform Model> Dasharo Release v0.x Signing Key
 sub   rsa4096 2021-11-10 [E] [expires: 2022-11-10]
 ```
+
+## Work with release signing key
+
+Three types of operations can be done with generated key. We can:
+
+* export public key for verification purposes,
+* use it to sign a release image,
+* use along others public keys to verify release image.
+
+### Exporting public key
+
+After receiving the new key, signed with the public part of the key that you
+have access to, please follow these steps to share the public part of new
+release key by adding it to this, 3mdeb-secpack, repository.
```

Comment:
@TomaszAIR we have to avoid ambiguous terms like: it, this, that. It is better to be redundant in description than risking misunderstanding.

Reply:
Will try to remember that. Here changed https://github.com/3mdeb/3mdeb-secpack/commit/9ccffe26e9b19df4f1345e05d854c749ddafd07a#diff-654059b91282a5de465b6a5377ddfb808c05bef732ba1e00c66b3664c61275faR82